cmd/relayproxy/helm-charts/relay-proxy/goff-proxy.yml

```diff
@@ -0,0 +1,9 @@
+---
+listen: 1031
+pollingInterval: 1000
+startWithRetrieverError: false
+retriever:
+  kind: http
```

Comment:
We plan to use GitHub retriever because we have standardized on GitHub as the single source repository, which use githubToken parameter to pass the token. We plan to use K8s secret to pass the token for obvious security reason, and need to replace the githubToken value with the provided K8s secret. It will be great to illustrate how to use K8s secret to provide credentials in the relay proxy config file, which is stored as configmap and mounted as a volume by this Helm chart.

Reply:
I also asked our DevOps engineers to review this PR tomorrow, and see if they have additional comments.